main/actor.hs

```diff
@@ -14,6 +14,8 @@ import Options.Generic
 data Args = Args
   { config  :: FilePath
     -- ^ Configuration file.
+  , storeconf :: Maybe Bool
```

Comment:
I think you can just do Bool here and don't the maybe - if the flag isn't set, it'll default to false. See `nocopy`.

Reply:
done, see [b3728a1](https://github.com/swift-nav/wolf/pull/52/commits/b3728a1beab3f3713beee221184d599706493382)